### PR TITLE
Add `.netlify` in .gitignore

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,6 @@ npm-debug.log
 .DS_Store
 .AppleDouble
 .LSOverride
+
+# Netlify
+.netlify
```
Comment (nschonni) [inline .gitignore:14]: Not really familiar with Netifly, but is this a file or folder that gets ignored?

Comment (XhmikosR) [reply .gitignore:14]: Yeah,

Comment (nschonni) [reply .gitignore:14]: Sorry, I think part of your response got cut off

Comment (nschonni) [reply .gitignore:14]: Oh, I guess my question might not have been clear. Is this a file or a folder? If it's a folder it should have a trailing slash

Comment (XhmikosR) [reply .gitignore:14]: It's a folder but due a bug in their client, it just adds `.netlify` in .gitignore even if one has `/.netlify/`

**Review by lpinca — APPROVED** [review]

**Pull request merged**